internal_docs/review-apps.md

```diff
@@ -0,0 +1,26 @@
+# Pull Request Review apps
+
+This repo uses [Heroku review apps](https://devcenter.heroku.com/articles/github-integration-review-apps)
+to give reviewers a way to easily preview and test changes to the app.
+
+If a pull request is created from a fork of this repo (typical if the
+contributor is not part of the alphagov organisation) then the review app will
+not be created automatically; however it can be created manually from the
+Heroku dashboard.
+
+Previews of Pull Requests are automatically published to a URL which has the
+prefix `govuk-prototype-kit` followed by the identifier number of the pull request.
+
+For example, pull request #137 would be deployed to `govuk-prototype-kit-pr-137.herokuapp.com`.
+
+The govuk-design-system-ci should create a GitHub deployment as Heroku builds
+and deploys the review app. The deployment will appear both as an event in the
+PR timeline and as an active deployment in the deployment status above the PR
+status checks. Once the review app has been deployed a button linking to the
+review app will become active.
+
+## Configuration
+
+The configuration of the review apps in this repo is maintained in the Design
+System Heroku account, the credentials for which can be found in the credential
+store.
```

Comment:
Should/could we link to the credential store from the word "credential store" here?

Reply:
The credential store is a private repo, I don't know whether we want to avoid leaking the name of the repo or not; what do you think @36degrees?

Reply:
I don't think we gain much from trying to keep it secret, but bear in mind anyone that doesn't have access will just get a 404 – is that going to cause confusion?

Reply:
> I don't think we gain much from trying to keep it secret, but bear in mind anyone that doesn't have access will just get a 404 – is that going to cause confusion?

I guess that it depends on whether it's clear enough that these are docs aimed mainly at team members than external people.

I've noticed that we also link to the credential store from https://github.com/alphagov/govuk-frontend/blob/master/docs/releasing/publishing.md:

> 13. Log into npm (`npm login`), using team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/master/npm/govuk-patterns-and-tools).

Reply:
I've added a link.